tests/test_utils.py

```
@@ -149,27 +208,47 @@ class DatabaseTestHelper:
     
     @staticmethod
     def create_temp_database():
-        """Create a temporary database file."""
+        """
+        Create and return the file path of a temporary database file.
+        
+        Returns:
+            str: The file path to the created temporary database file.
+        """
         temp_db = tempfile.NamedTemporaryFile(suffix='.db', delete=False)
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)